src/System.Linq.Expressions/tests/System.Linq.Expressions.Tests.csproj

```diff
@@ -16,7 +16,7 @@
     <DefineConstants Condition=" '$(IsInterpreting)' != 'true' ">$(DefineConstants);FEATURE_COMPILE</DefineConstants>
     <DefineConstants Condition=" '$(FeatureInterpret)' == 'true' ">$(DefineConstants);FEATURE_INTERPRET</DefineConstants>
     <KeepAllProjectReferences>true</KeepAllProjectReferences>
-    <NugetTargetMoniker>.NETStandard,Version=v1.6</NugetTargetMoniker>
+    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
I added the condition, but didn't add an extra configuration, since it was already being tested for netcoreapp1.0, nca1.1 and net463